### ISSUE TITLE
Add variable and functionality to assign runner to Runner Group during registration

### ISSUE THREAD

**Issue by drachkov** [terraform-google-modules/terraform-google-github-actions-runners#68]
### TL;DR

Based on current code level there is no possibility to specify a Runner Group for Enterprise organization accounts. Currently all registered new runners are automatically assigned to `Default` Runners Group.

Hence, the suggestion is to add an option to register runner and assign to the group(s) as part of this module for MIG native VMs and containers. 
Documentation upon groups can be found [here](https://docs.github.com/en/enterprise-cloud@latest/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups).

Thanks

### Terraform Resources

```markdown
# Add variable

variable "gh_runner_group" {
  type        = string
  description = "GitHub runner group"
  default     = "Default"
}
```

### Detailed design
The change would most likely need to happen in  startup scripts (VMs) and entrypoint.sh (containers) for `/runner/config.sh` parameters. Something like that:

```markdown

RUNNER_ALLOW_RUNASROOT=1 /runner/config.sh --unattended --replace --work "/runner-tmp" --url "$REPO_URL" --token "$ACTIONS_RUNNER_INPUT_TOKEN" --runnergroup "$ACTIONS_RUNNER_GROUP"

```

**Comment by drachkov** [comment]
Any update on this?
Thanks
